modules/sdk-core/src/bitgo/wallet/iWallet.ts

```diff
@@ -93,6 +93,7 @@ export interface PrebuildTransactionOptions {
   enableTokens?: TokenEnablement[];
   nonce?: string;
   preview?: boolean;
+  isTss?: boolean;
```

Comment:
Let's make this an extension like `BuildTokenEnablementOptions` for use in `wallet.ts`.